src/ui/components/Carousel/index.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+import NukaCarousel from 'nuka-carousel';
+import React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import { getDirection } from 'core/i18n/utils';
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+
+type PropTypes = {|
+  i18n: Object,
+  // eslint-disable-next-line no-undef
+  sections: Array<React$Element<*>>,
+|};
+
+export const CarouselBase = ({ i18n, sections }: PropTypes) => {
+  if (!sections) {
+    throw new Error('sections are required for a Carousel component');
+  }
+
+  return (
+    <Card className="Carousel">
+      <NukaCarousel
+        autoplay={false}
+        autoplayInterval={4000}
+        cellAlign={getDirection(i18n.lang) === 'ltr' ? 'left' : 'right'}
```

Comment:
ah that's cool! maybe we could reverse the list of `sections` and start from the most right element in RTL, what do you think? This would be perfect RTL experience.

Reply:
Ohhh, makes sense. Well... in the future we're gonna randomize them though (#3332). So maybe best to just wait until that "fixes" the ordering 😉

Reply:
wfm, *but* it should ideally start on the right side when RTL mode.

Reply:
Just tweaked it so the slider starts on the right... due to some issues with the slider I can't get the scroll element to start on the right exactly right but I'll explore that later.

Reply:
Yeah I see that. It starts on the right but scrolls on the left anyway. Can you open an issue for that so we don't forget?

Reply:
👍  Filed #3344